cibyl/sources/elasticsearch/api.py

```diff
@@ -136,6 +136,12 @@ def get_builds(self: object, **kwargs: Argument):
                 build_statuses = [status.upper()
                                   for status in
                                   kwargs.get('build_status').value]
+            build_numbers = []
```

Comment:
This can be deleted I think

Reply:
Since `--build-number` has been deleted, yes, we can remove it. I didn't realize to do this change from the moment it wasn't deleted of the `Job` class. Done.